README.md

```diff
@@ -31,6 +31,12 @@ Finally, run the automated tests with `doit tests`.
 
 For manual testing, you can start the images with `doit up`, however please refer to the next section for a production-ready local deployment of AiiDAlab with aiidalab-launch.
 
+### Test on different architectures
+
+Images are built for multiple architectures (amd64 and aarch64) during continuous integration and pushed to the GitHub Container Registry (ghcr.io).
+To run automated tests or manual test against those images, simply specify the registry with the aformentioned `up` and `tests` commands (e.g. `doit up --registry=ghcr.io/).
```

Comment:
I think we should add the ability to specify the image tag/sha, when running `doit up`. I am not sure this is implemented in `dodo.py`.

Reply:
Also, despite I am logged in, it didn't work for me:

```bash
$ docker login ghcr.io -u yakutovicha
Password:
Login Succeeded
$ doit up --registry=ghcr.io/
.  up
aiidalab Pulling
aiidalab Error
```

Reply:
Can you try to provide more information here? Maybe you can run the command manually to get more verbose output?
```console
REGISTRY=ghcr.io/ docker-compose up --detach
```

Reply:
Not much is shown.
```bash
REGISTRY=ghcr.io/ docker-compose up --detach
[+] Running 0/1
 ⠿ aiidalab Error                                                                                                                                                                                                                                                                                                                                                    0.8s
Error response from daemon: denied
```
It might be related to the token access rights. I will re-test this after lunch.

Reply:
Indeed, the issue was related to the token access rights.

The only issue that remains is this:


> I think we should add the ability to specify the image tag/sha, when running `doit up`. I am not sure this is implemented in `dodo.py`.

@csadorf can you please comment on this last point?

Reply:
This was implemented in https://github.com/aiidalab/aiidalab-docker-stack/pull/289 .